main.C

```diff
@@ -475,7 +475,6 @@ LOCALFUN TAG *ReadAndProcessApeHeader(fstream &input) {
   const string magic(ape._magic, 0, 8);
 
   if (magic != APE_MAGIC) {
-    Warning("file does not contain ape tag\n");
```

Comment:
Maybe keep this as Info?

The problem with changing the "UI" in this way is that some user might rely on it

Reply:
I've changed this to an Info, and also added Info's on successful file opens so we can determine which file this info applies to.